components/chart/multiple-axes.md

```diff
@@ -21,7 +21,7 @@ This article contains the following sections:
 	* [Numerical Charts](#numerical-charts)
 * [Choose Axis Position](#choose-axis-position)
 * [Examples](#examples)
-	* [Categorical Chart - Value Axes Examples](#categorical-chart---value-axes-examples)
+	* [Categorical Chart - Value Axes Examples](#categorical-chart-value-axes-examples)
```

Comment:
It looks like the algorithm for navigating to a heading has changed and some characters are ignored. Is there more information about this?

Reply:
Yes, the algorithm has been changed for some characters - some are ignored but some are additionally included (e.g. https://github.com/telerik/blazor-docs/pull/2476/files#diff-57dc4953244663e3f5151779ab49f1249b11d42b2d045cea757fe7a4c7673aadR55 - "Using a NuGet.Config file" where the dot is included). Do you mean that you'd like to get more information on the reasoning for this? Or, maybe some specific guidelines for the changes?

Reply:
The goal of the question was to ensure we have a resource for future docs changes.